bin/blueoak-server.js

```diff
@@ -3,10 +3,21 @@
  * Copyright (c) 2015-2016 PointSource, LLC.
  * MIT Licensed
  */
+var parseArgs = require('minimist');
 var server = require('../');
 
+// parse arguments
+var argv = parseArgs(process.argv.slice(2));
+
+// convert mocks from CSV into an array
+var mocks = argv.mocks || argv.m;
+if (mocks) {
+    mocks = mocks.split(',');
+}
+
 server.init({
-    appDir: process.cwd()
+    appDir: process.cwd(),
+    mocks: mocks
```

Comment:
Yeah, I know what you mean. My initial preference was to avoid modifying the main script, but I figured it doesn't add much overhead, and it might be nice to parse other CLI args in the future.

I took this approach for the simplicity and it lends itself well to Dockerization. With `ENTRYPOINT ["blueoak-server"]` arguments can be passed to the image at the command line more simply. For example, `docker run blueoak-server --mocks mock-service` instead of `docker run --entrypoint bin/bos-mock.js blueoak-server --mocks mock-service`